extending-valtimo/plugin/custom-plugin-definition.md

```diff
@@ -82,6 +82,57 @@ class TwitterPluginFactory(
 }
 ```
 
+### Plugin categories
```

Comment:
I think this is ok, with a good example, but i think we can add pieces of explanation per code block to explain what's going on.